src/components/PayinComfirmation.jsx

```diff
@@ -81,6 +81,23 @@ const PayinComfirmation = () => {
     </div>,
   ];
 
+  const [walletBalance, setWalletBalance] = useState(0);
+  useEffect(() => {
+    getUserWalletBalance()
+  .then((balance) => {
+    if (balance === "Error") {
+      alert("Error getting balance");
+    } else {
+      // alert(`Your balance is: ${balance} USDT`);
+      setWalletBalance(balance);
+    }
+  })
+  .catch((error) => {
+    console.error("Error:", error);
+  });
+  }, [checkBalance]);
+
+
```

Comment:
`          <a herf="" className="underline underline-offset-1 italic">
            MoonPay
          </a>`
Line 106 -- The herf attribute in the anchor tag should be corrected to href.

Reply:
Corrected